builder/src/version.rs

```diff
@@ -13,3 +17,14 @@ pub fn get_rom_version() -> u16 {
         | ((ROM_VERSION_MINOR & 0x1F) << 6)
         | (ROM_VERSION_PATCH & 0x3F)
 }
+
+// Runtime Version - 32 bits
+// [31:16] unused
```

Comment:
Should we use more of the bits here? We could probably do 16 bits for patch and then 8 bits each for major and minor.

ROM and FMC versions have to share a 32 bit register, but RT doesn't have the same constraint.

Reply:
Done